components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -250,7 +250,6 @@ class FennecMigrator private constructor(
         private var historyStorage: Lazy<PlacesHistoryStorage>? = null
         private var bookmarksStorage: Lazy<PlacesBookmarksStorage>? = null
         private var loginsStorage: Lazy<SyncableLoginsStorage>? = null
-        private var loginsStorageKey: String? = null
```

Comment:
Let's undo this and add it to the baseline file. I don't think we should touch any migration code until it's no longer being used.

Reply:
These were UnusedPrivateMembers. You sure we want to keep this?

Reply:
Yeah, I'd rather us leave migration code completely untouched. We've seen weird bugs at runtime in this layer before.